drivers/hmis/app/models/hmis/hud/custom_service.rb

```diff
@@ -38,6 +38,12 @@ def within_range?(range)
     date_provided.between?(range.begin, range.end)
   end
 
+  def display_name
+    return service_name if service_type.custom_service_category.name == service_name
```

Comment:
Could add a `has_one :custom_service_category, through: :custom_service_type` to reference the category directly